### DISCUSSION TITLE
How to install WSA

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/WSA#456] [General]
I am not that well versed in GitHub
I want to know how to install WSA on a computer

**Comment by HotCakeX** [comment]
You don't need GitHub to install WSA. It's in Microsoft Store.
https://apps.microsoft.com/detail/9NJHK44TTKSX

**Reply by ghost** [reply]
@HotCakeX I don't see a Microsoft store

**Comment by Darkhost1999** [comment]
https://support.microsoft.com/en-us/windows/install-mobile-apps-and-the-amazon-appstore-on-windows-f8d0abb5-44ad-47d8-b9fb-ad6b1459ff6c
